src/common/test-pipe.c

```diff
@@ -833,7 +833,7 @@ test_pty_shell (void)
 
   const gchar *argv[] = { "/bin/bash", "-i", NULL };
 
-  pipe = cockpit_pipe_pty (argv, NULL, NULL);
+  pipe = cockpit_pipe_pty (argv, NULL, NULL, 24, 80);
```

Comment:
Is there a way we can unit test:

 * Opening a channel with the right options.
 * Changing the options inside an already open channel.

I believe that spawning a shell from test-pipe and having the shell send out its ROWS and COLS environment variables after receiving some trigger input ... that would do the trick no?